config/config.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+    'secret': 'codeninjaresdocmang',
+    'db': 'mongodb://localhost/docs'
+};
```

Comment:
Wouldn't it be better to have a newline at the end of all files for consistency? This is not the only file missing it so if you decide to add the newlines look for the other files too.